src/NUnitFramework/testdata/AttributeThrowingExceptionFixture.cs

```diff
@@ -0,0 +1,87 @@
+// ***********************************************************************
```

Comment:
Even though the new file template hasn't been updated in the 3.13 branch, I think we've started using the new condensed file header for any new files in this branch too, to simplify when porting things to 4.0.
It should be copyable from another newer file, like `ArrayHelper`

https://github.com/nunit/nunit/blob/v3.13-dev/src/NUnitFramework/framework/Internal/ArrayHelper.cs#L1

Reply:
I wasn't sure what the procedure was for adding files to v3. Updated the headers.